### PR TITLE
fix slave output over buffersize

### PR DESCRIPTION
**Pull request by funnywwh**

### CODE REVIEW

File: webtty/webtty.go [hunk 1]
```diff
@@ -72,7 +72,12 @@ func (wt *WebTTY) Run(ctx context.Context) error {
 		errs <- func() error {
 			buffer := make([]byte, wt.bufferSize)
 			for {
-				n, err := wt.slave.Read(buffer)
+				//base64 length
+				effectiveBufferSize := wt.bufferSize - 1
+				//max raw data length
+				maxChunkSize := int(effectiveBufferSize/4) * 3
```
Comment (sorenisanerd) [inline webtty/webtty.go:78]: Oh, that's a good catch!

For compatibility reasons, we don't always use base64 encoding, so this should depend on the encoder being used.

**Review by sorenisanerd — CHANGES_REQUESTED** [review]
Oh, that's a good catch!

For compatibility reasons, we don't always use base64 encoding, so this should depend on the encoder being used.

**Comment by funnywwh** [comment]
> Oh, that's a good catch!
> 
> For compatibility reasons, we don't always use base64 encoding, so this should depend on the encoder being used.

So,how to do that?Close?

**Comment by sorenisanerd** [comment]
I'm sorry. I was wrong. I was thinking about when we talk to the master, not the slave.

Great work. Thanks :)

**Pull request merged**